### PR TITLE
LPS-128860 Upgrade report

### PR DESCRIPTION
**Pull request by achaparro**
Hi @tinatian,

I think we have included all changes requested. The implementation of the UpgradeReport class is minimal yet because we want to add it in the next ticket.

Let us know what you think.

Thanks!

cc/ @samziemer

### CODE REVIEW

**Comment by achaparro** [comment]
ci:test:sf

**Comment by achaparro** [comment]
ci:test:relevant

**Comment by achaparro** [comment]
ci:test:sf

**Comment by achaparro** [comment]
ci:test:relevant

File: modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/log/UpgradeLogAppender.java [hunk 1]
```diff
@@ -105,13 +103,15 @@ public boolean isStopped() {
...
 	}
 
-	public void setUpgradeReport(UpgradeReport upgradeReport) {
-		_upgradeReport = upgradeReport;
-	}
-
 	@Override
 	public void start() {
-		_started = true;
+		if (PropsValues.UPGRADE_REPORT_ENABLED) {
```
Comment (tinatian) [inline modules/apps/portal/portal-upgrade-impl/src/main/java/com/liferay/portal/upgrade/internal/log/UpgradeLogAppender.java]: @achaparro @SamZiemer 
Generally we do not use `PropsValues` directly in a component, use `Props` service by `@Reference` instead.
In your case, as you have a portal properties, and the entrance is in a portal-impl class `DBUpgrader`
, I would prefer to add the switch logic in `DBUpgrader`, use it to decide to create `ServiceDependencyManager` or not.

**Comment by achaparro** [comment]
Hi @tinatian, ok I wil take care of that and resend

**Comment by achaparro** [comment]
I will send another pull

**Pull request closed without merging**